<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение таблицы</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Название&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_roast">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Обжарка&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ground">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Молотый&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_taste">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Вкус&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Цена&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_volume">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Объем&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taste_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Вкус</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="roast_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="price_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="volume_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="ground_edit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Да</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Нет</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="new_data_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_data_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
